package.json

```diff
@@ -5,7 +5,7 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "tsc --build tsconfig.json",
+    "build": "rm -rf packages/*/build && tsc --build tsconfig.json",
```

Comment:
We need a script to clean all `node_modules` i experiences many problem with yarn symlink. And I prefer to have `clean build` script to clean `build`, `node_module` folders then install everything then build again.

Reply:
I add `clean-build` script